requirements.txt

```diff
@@ -3,7 +3,7 @@ alembic
 psycopg2
 numpy
 netCDF4
-nchelpers>=5.1.2
+nchelpers>=5.3.0
```

Comment:
I'd recommend [*pinning*](http://nvie.com/posts/pin-your-packages/) the version to 5.3.0 here. Or at the very least '==5'. Presumably, we don't want to auto-upgrade major versions which, [by definition](https://semver.org/), have backwards-incompatible API changes